src/components/services-view/services-view.controller.ts

```diff
@@ -48,29 +55,48 @@ export class ServicesViewController implements angular.IController {
     let categoryObj: any;
     let subCategoryObj: any;
 
+    if (subCategory ===  'null') {
```

Comment:
@dtaylor113 Why string `"null"` instead of the real JavaScript `null` value? Actually setting it to `null` seems cleaner to me.